### PR TITLE
*: update tpr status

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -224,6 +229,15 @@ func (c *Cluster) run(stopC <-chan struct{}, wg *sync.WaitGroup) {
 					return
 				}
 			}
+
+			if !reflect.DeepEqual(c.cluster.Status, c.status) {
+				newCluster := c.cluster
+				newCluster.Status = c.status
+				newCluster, err := k8sutil.UpdateClusterTPRObject(c.config.KubeCli, c.config.MasterHost, c.config.Namespace, newCluster)
```
Comment (hongchaodeng) [inline pkg/cluster/cluster.go]: c.config.Namespace -> c.cluster.Namespace

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -51,7 +52,10 @@ type clusterEvent struct {
 type Config struct {
 	PVProvisioner string
 	s3config.S3Context
-	KubeCli *unversioned.Client
+
+	MasterHost string
+	Namespace  string
```
Comment (hongchaodeng) [inline pkg/cluster/cluster.go]: Namespace already exist in "cluster" obj.

**Comment by hongchaodeng** [comment]
k8s cluster is down due to disk full

**Comment by hongchaodeng** [comment]
@etcd-bot retest this please

**Comment by hongchaodeng** [comment]
@etcd-bot retest this please

**Comment by hongchaodeng** [comment]
@etcd-bot retest this please

**Comment by hongchaodeng** [comment]
lgtm

**Comment by hongchaodeng** [comment]
@etcd-bot retest this please

**Comment by xiang90** [comment]
@hongchaodeng Tests are fixed. Merging.

**Pull request merged**